samples/nextjs/src/pages/api/track/[action].ts

```diff
@@ -0,0 +1,18 @@
+import { createProxyMiddleware } from 'http-proxy-middleware';
+
+import jssConfig from 'temp/config';
+
+export default createProxyMiddleware({
+  target: jssConfig.sitecoreApiHost,
+  changeOrigin: true,
+  pathRewrite: { '^/api': '/sitecore/api' },
+  headers: { sc_apikey: jssConfig.sitecoreApiKey },
+  xfwd: true,
```

Comment:
We need tracking requests to be sent for localhost because VisitorIdentification works this way.
Otherwise, we have two different sessions (cookies) on the server. The existing tracker does not track correctly because of it.

Url rewrite changes client IP but does not add x-forwarded-for. Here is the issue:
https://github.com/vercel/next.js/pull/17557

Url rewrite does not work in combination with /api/... (req, res are lost in this case).

I have not found any other easy way to pass the tracking request via localhost with the correct IP address.
I have not found out-of-the-box support of it in Next.js API feature.

Reply:
Resolving this one. As discussed, this seems to be our best option until Next.js supports this better OOTB.